libs/utils/android/workload.py

```diff
@@ -92,7 +92,7 @@ def availables(target):
                     module_name = module.__name__
                     # Check if a package is required and is available on target
                     aw = Workload._AW(module_name, module, class_name, handler)
-                    if (Workload._is_available(target, aw)):
```

Comment:
Was this broken since before this patch?!?
In that case we shoud better post this as a separate fix patch.